node/defaults.go

```diff
@@ -59,7 +59,7 @@ func DefaultDataDir() string {
 		if runtime.GOOS == "darwin" {
 			return filepath.Join(home, "Library", "Ethereum")
 		} else if runtime.GOOS == "windows" {
-			return filepath.Join(home, "AppData", "Roaming", "Ethereum")
+			return filepath.Join(os.Getenv("APPDATA"), "Ethereum")
```

Comment:
This change should not be made in the same PR. We need to test this extensively because it might ignore a previously-used data directory.

Reply:
ok! Please see `APPDATA` fix PR #17786 is separated now